app/controllers/users/confirmations_controller.rb

```diff
@@ -2,6 +2,12 @@ module Users
   class ConfirmationsController < Devise::ConfirmationsController
     include ValidEmailParameter
 
+    def create
+      super
+
+      UserMailer.already_confirmed(resource).deliver_later if resource.confirmed?
```

Comment:
No, you're right. I thought about this some more, and it's not immediately clear. I will add a comment. The `create` action is called when a user asks for confirmation instructions to be sent. If a user has already confirmed their email, but asks for instructions again, Devise will not send them an email by default, so we override the `create` method so that an email is sent letting the user know that they've already confirmed.